### DISCUSSION TITLE
Diagonal paths

### DISCUSSION THREAD

**Discussion by adammw** [OpenRCT2/OpenRCT2#19103] [Feature requests]
Diagonal paths look really silly.

![image](https://user-images.githubusercontent.com/153219/211507570-26a8256a-dc3a-47af-9fdc-82247fe352ce.png)

Would Like to have a render improvement to draw diagonal paths instead when there's only 1x1 path blocks like this, and have peeps walk diagonally along it.

**Comment by IntelOrca** [comment]
Without adding new graphics, how are you proposing this would be done?

**Comment by karst** [comment]
There's currently some custom paths available that can somewhat do this. ToonTowner has made a few paths that do this.
Examples to XXPATH02 and XXPATH07 (and probably things in between as well).
![image](https://user-images.githubusercontent.com/1516893/211547848-0d9567b0-a3c9-4071-92dd-f224c7496c3b.png)

Would be nice if support would be added to swap between different versions of path types, say the RCT1 and RCT2 path slopes (without having them be duplicates of one another) and having these diagonals available on the content that supports them if people want to use them.
